### PR TITLE
Reorg tool

### PR DESCRIPTION
**Pull request by pascalfleury**
Adding a tool to mechanically manage the Turtle files.
Currently the tool can do linting and plain merging.

### CODE REVIEW

**Review by alex-jansen — APPROVED** [review]

File: software/util/reorg.py [hunk 1]
```diff
@@ -0,0 +1,101 @@
+#!/usr/bin/env python
+"""A Tool to help maintain the set of ttl files in schema.org.
+
+This helps reformatting, merging and splitting ttl files along
+semantic axis.
+"""
+
+from collections.abc import Sequence
```
Comment (MatthiasWiesmann) [inline software/util/reorg.py:8]: Consider only including namespaces, not Classes.

File: software/util/reorg.py [hunk 2]
```diff
@@ -0,0 +1,101 @@
+#!/usr/bin/env python
```
Comment (MatthiasWiesmann) [inline software/util/reorg.py:1]: consider adding the emacs + encoding header.

#!/usr/bin/env python3
# -*- coding: UTF-8 -*-

File: software/util/reorg.py [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+semantic axis.
+"""
+
+from collections.abc import Sequence
+from rdflib import Graph, Namespace
+import logging
+import sys
+import argparse
+
+def SchemaOrgGraph():
```
Comment (MatthiasWiesmann) [inline software/util/reorg.py:14]: Consider adding type annotations as this is a new file, i.e. 

def SchemaOrgGraph() -> rdflib.Graph:

File: software/util/reorg.py [hunk 4]
```diff
@@ -0,0 +1,101 @@
...
+  only_in_g2 = g2 - g1
+  # TODO: print sampled diffs if any.
+  return len(only_in_g1) == 0 and len(only_in_g2) == 0
+
+def GraphIsIncludedIn(g_subset, g_universe):
+  only_in_subset = g_subset - g_universe
+  # TODO: print sampled diffs if any.
+  return len(only_in_subset) == 0
+
+def Lint(args):
```
Comment (MatthiasWiesmann) [inline software/util/reorg.py:31]: I would pass the arguments explicitly, i.e. output_filename and files so the function can be easily called from Python code/unit-tests.

File: software/util/reorg.py [hunk 5]
```diff
@@ -0,0 +1,101 @@
...
+  """Reformats the file(s) properly."""
+  def LintOne(filename, output_filename):
+    logging.info(" - reading file ...")
+    g = SchemaOrgGraph()
```
Comment (MatthiasWiesmann) [inline software/util/reorg.py:35]: Why g and v?  Maybe make the variable names longer / more expressive?

**Pull request merged**